### DISCUSSION TITLE
Have a "quickstart" section / more detailed installation info somewhere?

### DISCUSSION THREAD

**Discussion by ricalessandri** [SSAGESLabs/PySAGES#84] [Ideas]
I am not sure if this is more appropriate here or as an issue but I wanted to try to use the discussions :)

It could be useful to have a more detailed installation description somewhere. However, I don't know whether this should be part of (at least for the time being) the [main README](https://github.com/SSAGESLabs/PySAGES/blob/main/README.md) or should be put somewhere else.

What I have in mind is to have a bunch of commands that allow a new user to get everything up and running easily. Those might be platform dependent (Linux, macOS).

For example, currently in the main README there's only a link to `openmm-dlext` but no further info are provided on how to build that. Thanks to Pablo, I have a list of packages that need to be installed for example on a mac in order to build `openmm-dlext` successfully.

Shall I give a try to draft a "quickstart" sort of section somewhere?

Let me know what you think.

**Comment by p-zubieta** [comment]
Sorry I missed this earlier. These are great points and we actually are planing to tackling this. As a starting point we are going to collect the notes from everyone that has installed or has had issues installing pysages. The shared document is here https://hackmd.io/Jbpc1E2kRbKPLUnmLKwINA. Once we collect some info there, we'll use it to draft a PR on this. Feel free to head over there if you have some notes we can use.
